### PR TITLE
audit: enable audit during booting up

### PR DESCRIPTION
**Pull request by tjjh89017**
related to https://github.com/harvester/os2/pull/44

Signed-off-by: Date Huang <date.huang@suse.com>

### CODE REVIEW

File: package/harvester-os/files/etc/cos/bootargs.cfg [hunk 1]
```diff
@@ -2,7 +2,7 @@ set console_params="console=tty1"
 set kernel=/boot/vmlinuz
 set crash_kernel_params="crashkernel=219M,high crashkernel=72M,low"
 if [ -n "$recoverylabel" ]; then
-    set kernelcmd="$console_params root=live:LABEL=$recoverylabel rd.live.dir=/ rd.live.squashimg=$img panic=0 net.ifnames=1 rd.cos.oemtimeout=120"
+    set kernelcmd="$console_params root=live:LABEL=$recoverylabel rd.live.dir=/ rd.live.squashimg=$img panic=0 net.ifnames=1 rd.cos.oemtimeout=120 audit=1 audit_backlog_limit=8192"
```
Comment (bk201) [inline package/harvester-os/files/etc/cos/bootargs.cfg:5]: I guess we should add this in the else clause later. This only applies to recovery mode.

Comment (tjjh89017) [reply package/harvester-os/files/etc/cos/bootargs.cfg:5]: oh. I will fix this.

**Review by bk201 — APPROVED** [review]
LGTM

**Review by guangbochen — APPROVED** [review]
LGTM

**Review by Vicente-Cheng — APPROVED** [review]
LGTM, thanks!

**Pull request merged**